src/app/pages/SearchResultsPage/notifications.tsx

```diff
@@ -23,10 +22,9 @@ const NotificationBox = styled(Box)(({ theme }) => ({
 
   minHeight: 50,
 
-  background: theme.palette.layout.notificationBackground,
-  border: `2px solid ${theme.palette.layout.darkBorder}`,
-  color: theme.palette.layout.notificationText,
-
+  background: theme.palette.layout.secondary,
+  color: theme.palette.layout.titleOnBackground,
+  cursor: 'pointer',
```

Comment:
This is still bad accessibility https://github.com/oasisprotocol/explorer/pull/408#discussion_r1202752991